recipe/meta.yaml

```
@@ -17,21 +17,29 @@ build:
 
 requirements:
   host:
-    - python
+    - python  >=3.6
     - setuptools
     - jupyter
+    - jupyter-packaging
```

Maybe with this lower bound to ensure fixes from latest releases?

```suggestion
    - jupyter-packaging >=0.7.9
```